### ISSUE TITLE
Feature Request: add graph of the versions of each package

### ISSUE THREAD

**Issue by DeflateAwning** [crflynn/pypistats.org#70]
Feature Request: add graph of the versions of each package. 

Showing which version of my libraries each user is using is helpful is determining where to invest development effort on things like DeprecationWarnings. Currently, there doesn't seem to be a way to determine this distribution.

Let me know if this also requires a request on the upstream data fetch library.

**Comment by ktbarrett** [comment]
I agree. I would even like to see all the current data broken down by package version. For example, it'd be interesting to me to see what versions users who are still on Python 3.6 are installing.

We are debating dropping 3.6 support finally, but there are still Python 3.6 users. However, if all those 3.6 users are holding on to old package versions, it's doubtful they would want to upgrade to the next package version, so we can ignore them.

**Comment by DeflateAwning** [comment]
To add on, this is a feature that Rust's crates.io shows, and it's very helpful. It's too bad the Python ecosystem is lagging in this regard
